ansible/roles/verifiable-logger/templates/verify-logs.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=verify-logs
+
+[Service]
+ExecStart=/usr/local/bin/verifiable-logger verify file-to-s3-reliability-test-data us-east-1 --prefix "host=%H" --tail
```

Comment:
This is pretty hardcoded right now, but should obviously be templated in once we figure out the right way to get the variables in here.

Reply:
I would highly recommend not adding support for environment variables in Systemd. I also ran into weird issues when I passed more than one flag. It was a mess. But if you look at the `http_test_server` role I used a template for that service file.